modules/multi-architecture-creating-x86_64-bootimage.adoc

```diff
@@ -0,0 +1,135 @@
+----
++
+<1> The `westus` object is an example region.
++
+. Create a storage container using the storage account you generated by running the following command:
++
+[source,terminal]
++
+----
+$ az storage container create -n ${CONTAINER_NAME} --account-name ${STORAGE_ACCOUNT_NAME}
+----
+. You must use the {product-title} installation program JSON file to extract the URL and `x86_64` VHD name:
+.. Extract the `URL` field and set it to `RHCOS_VHD_ORIGIN_URL` as the file name by running the following command:
++
+[source,terminal]
+----
+$ RHCOS_VHD_ORIGIN_URL=$(oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' | jq -r '.architectures.x86_64."rhel-coreos-extensions"."azure-disk".url')
+----
+.. Extract the `x86_64` VHD name and set it to `BLOB_NAME` as the file name by running the following command:
++
+[source,terminal]
+----
+$ BLOB_NAME=rhcos-$(oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' | jq -r '.architectures.x86_64."rhel-coreos-extensions"."azure-disk".release')-azure.x86_64.vhd
+----
+. Generate a shared access signature (SAS) token. Use this token to upload the {op-system} VHD to your storage container by running the following commands:
++
+[source,terminal]
+----
+$ end=`date -u -d "30 minutes" '+%Y-%m-%dT%H:%MZ'`
+----
++
+[source,terminal]
+----
+$ sas=`az storage container generate-sas -n ${CONTAINER_NAME} --account-name ${STORAGE_ACCOUNT_NAME} --https-only --permissions dlrw --expiry $end -o tsv`
+----
+. Copy the {op-system} VHD into the storage container by running the following command:
++
+[source,terminal]
+----
+$ az storage blob copy start --account-name ${STORAGE_ACCOUNT_NAME} --sas-token "$sas" \
+ --source-uri "${RHCOS_VHD_ORIGIN_URL}" \
+ --destination-blob "${BLOB_NAME}" --destination-container ${CONTAINER_NAME}
+----
++
+You can check the status of the copying process by running the following command:
++
+[source,terminal]
+----
+$ az storage blob show -c ${CONTAINER_NAME} -n ${BLOB_NAME} --account-name ${STORAGE_ACCOUNT_NAME} | jq .properties.copy
+----
++
+.Example output
+[source,terminal]
+----
+{
+ "completionTime": null,
+ "destinationSnapshot": null,
+ "id": "1fd97630-03ca-489a-8c4e-cfe839c9627d",
+ "incrementalCopy": null,
+ "progress": "17179869696/17179869696",
+ "source": "https://rhcos.blob.core.windows.net/imagebucket/rhcos-411.86.202207130959-0-azure.aarch64.vhd",
+ "status": "success", <1>
+ "statusDescription": null
+}
+----
++
+<1> If the `status` parameter displays the `success` object, the copying process is complete.
+
+. Create an image gallery by running the following command:
++
+[source,terminal]
+----
+$ az sig create --resource-group ${RESOURCE_GROUP} --gallery-name ${GALLERY_NAME}
+----
+Use the image gallery to create an image definition. In the following example command, `rhcos-x86_64` is the name of the image definition.
```

Comment:
> `rhcos-x86_64` is the name of the image definition.
Should this be a call out <1> in the command?

If so, 
```suggestion
Use the image gallery to create an image definition by running the following command:
```

Reply:
Updated as per the suggestion.